.github/workflows/spelling.yml

```diff
@@ -62,3 +70,9 @@ jobs:
           extra-dictionaries: ${{ inputs.extra-dictionaries }}
           config: ${{ inputs.config }}
           spell-check-this: ${{ inputs.spell-check-this }}
+
+      - name: Cancel Workflow on Failure
```

Comment:
> [Document this](https://github.com/Lombiq/GitHub-Actions/blob/dev/Docs/Workflows.md#spelling-workflow) as the recommended setup for Spelling when used as an independent job in a larger workflow.

was not addressed.

I see the `cancel-workflow` action and the `cancel-workflow-on-failure` are well documented, but noting this behavior under not just the `spelling` but all affected workflow documentation would be necessary.

Reply:
I didn't do that because while initially, I thought this will be an optional setup for just Spelling that you as a consumer of these workflows should do, but then this became the default behavior for most workflows. But a mention of this under workflows is indeed necessary.